### DISCUSSION TITLE
Emotion uses this template literal notation.  Is it possible with Theme UI?

### DISCUSSION THREAD

**Discussion by dungle-scrubs** [system-ui/theme-ui#1312] [Q&A] [answered]
My understanding is that Theme UI is a wrapper around Emotion.  With Emotion, you can do this:

```
import styled from '@emotion/styled'

const Button = styled.button`
  color: turquoise;
`

render(<Button>This my button component.</Button>)
```

However, I can't find a similar way of wrapping a component into another styled component and the above syntax doesn't work when importing `Styled` from Theme UI

Thoughts?

**Comment by lachlanjc** [comment, marked as answer]
Yep! You’ve got it exactly. Theme UI is a layer on top of Emotion, but @emotion/styled is a dependency, so just use `styled` directly like that. If you’d like to style a component, `styled(Badge)` etc will work. I use it [in production](https://github.com/hackclub/summer-site/blob/50b97e457f33c7d3f853d42478ab0d90e0f68ece/components/flag.js#L2)  frequently. Theme UI’s `Styled` is a totally different thing, a React component for using theme `styles.` variants.

**Comment by dungle-scrubs** [comment]
Thanks, I understand now.  For anyone else reading, I'll paraphrase my confusion :)

- Styled (capital "S") is a theme-ui import which provides a different function.

- styled (lowercase "s") is an @emotion import, which is a dependency of theme-ui.  However, to use it, you still have to import it from `@emotion/styled` but you don't have to add it to your package.json.

Thanks for the help

**Reply by hasparus** [reply]
Thanks @dungle-scrubs. Actually, do you have an idea for a rename? I agree that capitalized `Styled` might be confusing.

![image](https://user-images.githubusercontent.com/15332326/100601319-b97f2200-3302-11eb-9a2f-cd79d785a97a.png)

cc @lachlanjc

**Comment by dungle-scrubs** [comment]
> Actually, do you have an idea for a rename?

What about "Themed"? Not sure if it's being used or would cause confusion, but in my mind it's still close to describing its purpose.

**Reply by lachlanjc** [reply]
Yep, this came to mind for me too @hasparus. I never use the component personally, but I think it makes a lot more sense. I’m not sure how we’re handling breaking changes pre-1.0, or maybe we hold the change to 1.0?

**Reply by hasparus** [reply]
> I’m not sure how we’re handling breaking changes pre-1.0, or maybe we hold the change to 1.0?

We'll bump minor.

**Reply by hasparus** [reply]
I'm actually always renaming this component to `s` on import to write `<s.p>` `<s.h1>`.
`Themed` looks good IMO.
